### PR TITLE
Avoid naming collisions with brooklyn-hyperledger

### PR DESCRIPTION
**Pull request by mikezaccardo**

### CODE REVIEW

File: catalog/hyperledger/kubernetes.bom [hunk 1]
```diff
@@ -77,26 +77,26 @@ brooklyn.catalog:
             default: "info"
 
         brooklyn.children:
-          - type: hyperledger-namespace
-            id: hyperledger-namespace
+          - type: hyperledger-kubernetes-namespace
```
Comment (grkvlt) [inline catalog/hyperledger/kubernetes.bom]: Bit keen; what other type of namespace would we have :frog:

File: catalog/hyperledger/kubernetes.bom [hunk 2]
```diff
@@ -116,17 +116,17 @@ brooklyn.catalog:
                 - "wu3F1EwJWHvQ"
 
         brooklyn.children:
-          - type: hyperledger-membersrvc-pod
-            id: hyperledger-membersrvc-pod
+          - type: hyperledger-kubernetes-membersrvc-pod
```
Comment (grkvlt) [inline catalog/hyperledger/kubernetes.bom]: Ditto; only Kubernetes has Pods... Same for other Kubernetes specifics...

**Review by grkvlt — CHANGES_REQUESTED** [review]
Keep changed catalog entry names and ids for`hyperledger-kubernetes-vp-cluster`, `hyperledger-kubernetes-membersrvc`, `hyperledger-kubernetes-cli`, `hyperledger-kubernetes-validating-peer` and `hyperledger-kubernetes-root-validating-peer` *only*

**Comment by mikezaccardo** [comment]
@grkvlt I amended the commit to include name changes for only the entities you listed above.

**Pull request merged**